apps/files/src/components/FilesAppBar.vue

```diff
@@ -525,14 +525,18 @@ export default {
     },
 
     $_ocTrashbin_deleteSelected () {
-      const translated = this.$gettext('%{numberOfFiles} items will be deleted immediately. You can’t undo this action.')
-      this.setTrashbinDeleteMessage(this.$gettextInterpolate(translated, { numberOfFiles: this.selectedFiles.length }, true))
+      const translated = this.$ngettext(
+        '%{numberOfFiles} item will be deleted immediately. You can’t undo this action.',
```

Comment:
Mind also fixing that suspicious apostrophe that looks like a forward tick ?
You can also change it to "cannot"